ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/confirmation/clientlibs/js/confirmation.js

```diff
@@ -5,9 +5,15 @@ $(document).ready(function () {
     const getPorductDetails = JSON.parse(localStorage.getItem('productDescription'));
     const checkOutDeatils = JSON.parse(localStorage.getItem('checkOutDetails')); 
 
+
+
+
+    // const getPorductDetails = {"cartdetails":[{"cartItem":{"image":"https://www.hcltech.com/sites/default/files/styles/large/public/images/guideline_based1.png","title":"Joust Duffle Bag","qty":"3","price":"34"}},{"cartItem":{"image":"https://www.hcltech.com/sites/default/files/styles/large/public/images/guideline_based1.png","title":"Strive Shoulder Pack","qty":"5","price":"32"}}],"fprice":"359.60","coupondiscount":"52.40","delivercharges":"150"};
```

Comment:
@AyushHCL : can we remove this comment block

Reply:
sure Rishab